source/_integrations/asuswrt.markdown

```diff
@@ -42,9 +42,18 @@ These sensors are automatically created and associated to the router device:
 - Upload sensor (unit_of_measurement: Gigabyte - *Daily accumulation*)
 - Upload Speed sensor (unit_of_measurement: Mbit/s)
 - Load average sensors (1min, 5min, 15min)
-- Temperature sensors (2ghz, 5ghz, cpu). NB: only temperature sensors available on your router will be created
+- Temperature sensors (2 GHz, 5 GHz, 6 GHz, CPU). Only temperature sensors available on your router will be created.
 
-Only `Connected devices sensor` is created in status **enabled**, all other sensors are created in status **disabled**. To use them, simply **enable** on the devices page.
+If the integration is configured to use the http(s) protocol, also the following sensors will be available:
+
+- CPU usage sensors (percentage for total and single core)
+- Memory usage sensor (percentage)
+- Free memory sensor (Megabyte)
+- Memory used sensor (Megabyte)
+- Last boot sensor (Timestamp)
+- Uptime sensor (Seconds)
```

Comment:
seconds as in hours, minutes, and seconds? Or seconds as in it will show 68400 if it's been running for a day?

Reply:
Not clear to me if this should be what HA interface show or how sensor is defined. Sensor definition say `native_unit_of_measurement=UnitOfTime.SECONDS` 

and this is the information in HA sensor properties:

![image](https://github.com/user-attachments/assets/883d80a5-270f-4f9a-8da3-3264c97b85fa)


But in the HA UI is shown as HH:MM:SS

![image](https://github.com/user-attachments/assets/bdc1dcc0-22e1-475e-ac13-5158d05014c2)

(Just reboot my router about 40 minutes ago)

Reply:
ah, ok, sorry. I meant what the users sees. so hours, minutes, and seconds (HH:MM.SS).

Reply:
Ok, changed as requested